src/util.rs

```diff
@@ -0,0 +1,92 @@
+use std::mem::transmute;
+
+use ff::PrimeField;
+use ruint::aliases::U256;
+
+/// Assumes little endian
+pub fn u256_as_limbs(uint: &U256) -> &[u32; 8] {
+    let limbs = uint.as_limbs();
+    unsafe { transmute(limbs) }
+}
+
+/// Assumes little endian
+pub fn limbs_as_u256(limbs: [u32; 8]) -> U256 {
+    let limbs: [u64; 4] = unsafe { transmute(limbs) };
+    U256::from_limbs(limbs)
+}
+
+/// Assumes little endian
+pub fn ff_as_limbs<F: PrimeField>(f: &F) -> &[u32; 8] {
+    let binding = f.to_repr();
+    let repr: &[u8; 32] = binding.as_ref().try_into().unwrap();
+    // this doesn't work if the platform we're on is not little endian :scream:
```

Comment:
It's even a bit worse than that: it assumes that the field implementation is using LE for `to_repr()`, see the tests introduced in https://github.com/lurk-lab/arecibo/pull/353/commits/fd3ae7a5e989ab62e6c128f79f5ebfae972f7fac

Reply:
Yes, should I be more explicit in mentioning this issue? There's also no way to directly test within `circom-scotia`, since we can't guarantee downstream will use any particular field impl... soooo

Reply:
I guess we have two ways of addressing this:
1. Keep this as a limitation for now, document it more clearly and open an issue
2. Change the trait we deal with to `PrimeFieldBits` instead of `PrimeField` to allow for `to_le_bits` method call, ensuring correctness.

@Hanting Zhang, feel free to chose any.